packages/wp-story-editor/src/api/getStory.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@web-stories-wp/design-system';
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+// Important: Keep in sync with REST API preloading definition.
+const STORY_FIELDS = [
+  'id',
+  'title',
+  'status',
+  'slug',
+  'date',
+  'modified',
+  'excerpt',
+  'link',
+  'story_data',
+  'preview_link',
+  'edit_link',
+  'embed_post_link',
+  'publisher_logo_url',
+  'permalink_template',
+  'style_presets',
+  'password',
+].join(',');
+
```

Comment:
See this line will need to added 

https://github.com/google/web-stories-wp/blob/9d18d5f52f1b8a88f170df6be099b0a9d69e40c2/packages/story-editor/src/app/api/apiProvider.js#L58 now that https://github.com/google/web-stories-wp/pull/8946 is merged.

Reply:
Done, thanks